### DISCUSSION TITLE
Docker deployment: Services documentation

### DISCUSSION THREAD

**Discussion by Nautigsam** [appwrite/appwrite#1066] [Q&A] [answered]
I use the docker-compose example setup and I am surprised by the quantity of services to deploy. I understand that some are optional like ClamAV, but it is difficult to know what are these services used for, which are optional or not, which env variables they require, etc.

Is there a documentation somewhere that could summarize all this info of is it yet to be done?

**Comment by eldadfux** [comment, marked as answer]
Actually this is a work in progress I already started, will try to push and finish it this week.

**Comment by eldadfux** [comment]
@Nautigsam and you can also take a look at "Appwrite Lite" for a simpler setup: https://github.com/appwrite/lite

**Comment by alexweininger** [comment]
Totally agree, I think MANY users of Appwrite will fall into the category that might know how to code, but might not know Docker, Traefik, reverse proxies, backend engineering, networking, etc. So in my opinion we can never have enough documentation on this materials. Especially because it will give users a lot more confidence in Appwrite even _before_ they decide to use it. Since quality and amount of documentation is something that many developers will be looking for when making a big choice like "should I use Appwrite or Firebase".
